left_comp:
  '-1': '0.0164883290091'
  '-10': '0.0167353824627'
  '-11': '0.0167775899722'
  '-12': '0.0168224016863'
  '-13': '0.0168697630948'
  '-14': '0.0169196291433'
  '-15': '0.0169719632318'
  '-16': '0.0170267363588'
  '-17': '0.0170839263893'
  '-18': '0.017143517428'
  '-19': '0.0172054992817'
  '-2': '0.0165022941346'
  '-20': '0.0172698669989'
  '-21': '0.0173366204751'
  '-22': '0.0174057641154'
  '-23': '0.0174773065466'
  '-24': '0.0175512603722'
  '-25': '0.0176276419654'
  '-26': '0.0177064712955'
  '-27': '0.0177877717832'
  '-28': '0.0178715701825'
  '-29': '0.017957896486'
  '-3': '0.0165200655382'
  '-30': '0.0180467838511'
  '-4': '0.0165414144972'
  '-5': '0.0165661446999'
  '-6': '0.016594087549'
  '-7': '0.0166250982644'
  '-8': '0.0166590526347'
  '-9': '0.016695844299'
  '0': '0.0164784369805'
  '1': '0.0164729298816'
  '10': '0.0167098107491'
  '11': '0.0167850374732'
  '12': '0.0168759578117'
  '13': '0.0169853861657'
  '14': '0.0171168557731'
  '15': '0.0172748471318'
  '16': '0.0174651051091'
  '17': '0.0176950861469'
  '18': '0.0179746000253'
  '19': '0.0183167489628'
  '2': '0.0164721730198'
  '20': '0.0187393324175'
  '21': '0.0192670021059'
  '22': '0.0199346657582'
  '23': '0.0207930511072'
  '24': '0.0219181836864'
  '25': '0.0234283663288'
  '26': '0.0255165788486'
  '27': '0.0285175142123'
  '28': '0.0330619524862'
  '29': '0.040488865271'
  '3': '0.0164765955845'
  '4': '0.0164867035244'
  '5': '0.0165030955835'
  '6': '0.0165264834052'
  '7': '0.0165577169199'
  '8': '0.0165978166521'
  '9': '0.0166480151713'
right_comp:
  '-1': '0.0164784369805'
  '-10': '0.0166480151713'
  '-11': '0.0167098107491'
  '-12': '0.0167850374732'
  '-13': '0.0168759578117'
  '-14': '0.0169853861657'
  '-15': '0.0171168557731'
  '-16': '0.0172748471318'
  '-17': '0.0174651051091'
  '-18': '0.0176950861469'
  '-19': '0.0179746000253'
  '-2': '0.0164729298816'
  '-20': '0.0183167489628'
  '-21': '0.0187393324175'
  '-22': '0.0192670021059'
  '-23': '0.0199346657582'
  '-24': '0.0207930511072'
  '-25': '0.0219181836864'
  '-26': '0.0234283663288'
  '-27': '0.0255165788486'
  '-28': '0.0285175142123'
  '-29': '0.0330619524862'
  '-3': '0.0164721730198'
  '-30': '0.040488865271'
  '-4': '0.0164765955845'
  '-5': '0.0164867035244'
  '-6': '0.0165030955835'
  '-7': '0.0165264834052'
  '-8': '0.0165577169199'
  '-9': '0.0165978166521'
  '0': '0.0164883290091'
  '1': '0.0165022941346'
  '10': '0.0167775899722'
  '11': '0.0168224016863'
  '12': '0.0168697630948'
  '13': '0.0169196291433'
  '14': '0.0169719632318'
  '15': '0.0170267363588'
  '16': '0.0170839263893'
  '17': '0.017143517428'
  '18': '0.0172054992817'
  '19': '0.0172698669989'
  '2': '0.0165200655382'
  '20': '0.0173366204751'
  '21': '0.0174057641154'
  '22': '0.0174773065466'
  '23': '0.0175512603722'
  '24': '0.0176276419654'
  '25': '0.0177064712955'
  '26': '0.0177877717832'
  '27': '0.0178715701825'
  '28': '0.017957896486'
  '29': '0.0180467838511'
  '3': '0.0165414144972'
  '4': '0.0165661446999'
  '5': '0.016594087549'
  '6': '0.0166250982644'
  '7': '0.0166590526347'
  '8': '0.016695844299'
  '9': '0.0167353824627'
